README.md

```diff
@@ -1,63 +1,99 @@
 # ManySecured-D3DB
 
-A repo for the ManySecured D3 data management pipeline from claims and behaviours.
+A repo for the management and storage of ManySecured D3 claims.
+
+## Index
+
+1. [D3 Architecture Summary](#d3-architecture-summary)
+2. [ManySecured Working Group Documentation and Specifications](#manysecured-working-group-documentation-and-specifications)
+3. [How to add a new D3 claim](#how-to-add-a-new-d3-claim)
+4. [Project Details](#project-details)
+   1. [Repo Folder Structure](#repo-folder-structure)
+   2. [D3 Claim Workflow](#d3-claim-workflow)
+
+## ManySecured Working Group Documentation and Specifications
 
-Links:
 - [ManySecured Docs](https://specs.manysecured.net/)
 - [ManySecured Docs: D3](https://specs.manysecured.net/d3)
 - [ManySecured Docs: D3 Claims](https://specs.manysecured.net/d3/D3%20claims)
 - [ManySecured Docs: D3 Behaviours](https://specs.manysecured.net/d3/D3%20behaviourss)
 
+## D3 Architecture Summary
+
+![Relationship Graph for the D3 Claim Types](./examples/D3-claim-dep-graph.png)
+
+### D3 Claim Type Summaries
+
+#### Device TYPE assertions
+
+Device types are the core description of an instance of a given device. The type assertion encapsulates the device's details and refers (by GUID reference) to the behaviours, vulnerabilities, and firmware associated with that device. Type assertions operate an inheritance model, where a type assertion can inherit the properties from parent types and then overload/add properties unique to that device type instance.
```

Comment:
It doesn't need to be in this PR, but it would be nice to have links to examples.

E.g. a link to https://specs.manysecured.net/d3/D3%20claims#assert-device-type or to something in the [`./examples` folder](https://github.com/TechWorksHub/ManySecured-D3DB/tree/main/examples).